imports/ui/unverified-warning/unverified-warning.jsx

```diff
@@ -27,12 +49,16 @@ UnverifiedWarning.propTypes = {
 }
 
 export default connect(
-  () => ({}) // Redux store to props
+  (resendVerificationState) => ({
```

Comment:
You need "destructure" the store object to get `resendVerificationState` out of it (just wrap it in curly brackets)